src/routes/api/assistant/[id]/embed-snippet/+server.ts

```diff
@@ -0,0 +1,160 @@
+export async function GET({ params }) {
+	const { id } = params;
+
+	const script = `(function() {
+	function resizeIframeToContentSize(iframe) {
```

Comment:
nice, i'm curious if there's a way to do this that's more maintainable ? it works great but i can imagine it would be very difficult to modify in the future

i know there's a `render()` method from svelte you can use to get the HTML for a component, we use something similar in thumbnails (https://github.com/huggingface/chat-ui/blob/c544c28d603fb40f74dec5b5271fde97f73b3fab/src/routes/assistant/%5BassistantId%5D/thumbnail.png/%2Bserver.ts#L44-L55) not sure how it handles javascript and idk if it's worth it but it could let you build this feature in a svelte component

Reply:
or simpler, make the JS snippet completely static (a static file) that gets the assistant's id from a property in a `<a>` link inside the embedding webpage

Reply:
```html
<script src="https://link/to/static.js"></script>

<a huggingchat-assistant="66017fca" href="https://hf.co/chat/assistant/66017fca">Chat with {Assistant}</a>
```